### PR TITLE
Shortcut Clarification

### PR DESCRIPTION
**Pull request by travisgosselin**
This updated removes the acceptable behavior of using conflicting paths for shortcuts by removing hierarchy IDs between nested resources. It also provides further clarification on shorcutting resources with inherit or default filters from the hierarchy.

### CODE REVIEW

File: standards/url-structure.md
Comment (eggilbert) [inline standards/url-structure.md:199]: Is this distinction here that there is an ID somewhere in the middle of the path instead of the end, which was leaning on or leaking that the ID was globally unique?

**Review by eggilbert — APPROVED** [review]

File: standards/url-structure.md
Comment (travisgosselin) [reply standards/url-structure.md:199]: Negative, distinction is simply that the shortcut applies to a hierarchy that is automatically filtered based on the path URL. But the usage in this example of "comments/authors" together is confusing. In fact I don't think we need the second example at all as it doesn't provide anything relevant.

Comment (travisgosselin) [reply standards/url-structure.md]: FYI - I removed the second example now .

**Comment by travisgosselin** [comment]
@alexander-ivakhnenko I know you had some thoughts on this. Will wait for your feedback before merging.

**Pull request merged**
